Assets/Scripts/Tracker/PoseTracker.cs

```diff
@@ -0,0 +1,102 @@
+// Copyright 2021-2022 The SeedV Lab.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Collections;
+using System.Collections.Generic;
+using Logger = Mediapipe.Logger;  // Disambiguation with UnityEngine.Logger
+using Mediapipe;
+using Mediapipe.Unity;
+using Mediapipe.Unity.PoseTracking;
+using UnityEngine;
+
+[RequireComponent(typeof(PoseTrackingGraph))]
+public class PoseTracker : MonoBehaviour {
+  private const string _TAG = nameof(PoseTracker);
+  public enum InferenceMode {
+    CPU,
+    GPU
```

Comment:
```suggestion
    GPU,
```

Google style suggests that every single line of a list-like struct ends with a comma. Example code from the style guide:

```
  public enum MyEnum {
    Yes,
    No,
  }
```